src/header/js/header-inline-title.js

```diff
@@ -0,0 +1,170 @@
+import {$assign as $, $dispatch, $replaceAll, $renderSvgOrImg} from '../../shared/js/shared';
+import {$close, $check, $pencil} from '../../shared/js/iconPaths.js';
+
+const prefix = 'esri-header-inline-title';
+const titleState = {};
+
+export default () => {
+	/* Title: Control
+	/* ====================================================================== */
+
+	const $control = $('button',
+		{
+			class: `${prefix}-control`, id: `${prefix}-control`,
+			tabindex: "-1",
+			aria: {expanded: false, controls: `${prefix}-content`}
+		}
+	);
+
+	$control.addEventListener('header:menu:open', (event) => {
+		$dispatch($control, 'header:inlineTitle:activated', {event});
+	});
+
+	$control.addEventListener('click', (event) => {
+		$dispatch($control, 'header:click:inlineTitle', {event});
+		$dispatch($control, 'header:menu:toggle', {
+			state: 'menu',
+			target: $target,
+			type: 'inlineTitle',
+			control: $control,
+			content: $content,
+			event
+		});
+	});
+
+	/* Title: Deactivates and Closes input
+	/* ====================================================================== */
+
+	const deactivateInput = () => {
+		$dispatch($control, 'header:inlineTitle:deactivated', {event});
+
+		setTimeout(() => {
+			$control.focus();
+		}, 0);
+
```

Comment:
this seems weird, why do you need the setTimeout?

Reply:
I had an issue with inline search where it wouldn't focus on control until `display: none` was removed. I might not need it here. I'll test it out and see.